carma/launch/guidance.launch.py

```diff
@@ -316,6 +316,59 @@ def generate_launch_description():
         ]
     )
 
+    trajectory_follower_container = ComposableNodeContainer(
+        package='carma_ros2_utils',
+        name='trajectory_follower_container',
+        executable='carma_component_container_mt',
+        namespace=GetCurrentNamespace(),
+        composable_node_descriptions=[
+            ComposableNode(
+                package='trajectory_follower_nodes',
+                plugin='autoware::motion::control::trajectory_follower_nodes::LatLonMuxer',
+                name='latlon_muxer_node',
+                extra_arguments=[
+                    {'use_intra_process_comms': False},
+                    {'--log-level' : GetLogLevel('latlon_muxer', env_log_levels) }
+                ],
+                parameters=[
+                    {'timeout_thr_sec':0.5}
+                ]
+            ),
+            ComposableNode(
+                package='trajectory_follower_nodes',
+                plugin='autoware::motion::control::trajectory_follower_nodes::LateralController',
+                name='lateral_controller_node',
+                extra_arguments=[
+                    {'use_intra_process_comms': True},
+                    {'--log-level' : GetLogLevel('lateral_controller', env_log_levels) }
+                ],
+                remappings = [
+                      ("output/lateral/control_cmd", "input/lateral/control_cmd")
+                ],
+                parameters = [
+                    [vehicle_calibration_dir, "/mpc_follower/lateral_controller_defaults.yaml"]
+                ]
+            ),
+            ComposableNode(
+                package='trajectory_follower_nodes',
+                plugin='autoware::motion::control::trajectory_follower_nodes::LongitudinalController',
+                name='longitudinal_controller_node',
+                extra_arguments=[
+                    {'use_intra_process_comms': False},
+                    {'--log-level' : GetLogLevel('longitudinal_controller', env_log_levels) }
+                ],
+                remappings = [
+                      ("output/longitudinal/control_cmd", "input/longitudinal/control_cmd"),
+                      ("input/current_trajectory", "input/reference_trajectory"),
+                      ("input/current_state", "input/current_kinematic_state")
+                ],
+                parameters = [
+                    [vehicle_calibration_dir, "/mpc_follower/longitudinal_controller_defaults.yaml"]
+                ]
+            )
+        ]
+    )
+
```

Comment:
I think this might be better to be in the plugins.launch.py close to its "main" node trajectory_follower_wrapper

Reply:
I dont have a problem with that, but it seemed to me that the pluginns.launch.py has all the carma plugins that we developed. While external autoware nodes such as twist filter are in guidance.launch.py
What do you think?

Reply:
It just seems easier if all the related nodes were in one file. Not a blocker though.